packages/notification/src/_styles.scss

```diff
@@ -328,22 +328,6 @@ $notification-slide-right: transform 300ms ease-out;
   }
 }
 
-%ca-notification__cancel__inner {
```

Comment:
This was only used in the Generic Notification component and is what was causing the misalignment